examples/air-quality-sensor-app/air-quality-sensor-common/air-quality-sensor-app.zap

```diff
@@ -3243,7 +3243,7 @@
               "code": 2,
               "mfgCode": null,
               "side": "server",
-              "type": "int16u",
+              "type": "vendor_id",
```

Comment:
These look like changes unrelated to this PR. If this is from running zap_convert_all.py, could you please do a separate PR for that?